UI/Marker/GenericPlacemark.gd

```diff
@@ -2,6 +2,8 @@ extends Button
 class_name GenericPlacemark
 
 signal placemark_removed()
+signal placemark_selected
+signal placemark_deselected
```

Comment:
If these signals are only used internally you could make and call a virtual (abstract) function instead, each subclass would have their own implementation if they need it.